app/src/main/java/com/cornellappdev/android/eatery/model/DiningHallModel.java

```diff
@@ -0,0 +1,290 @@
+package com.cornellappdev.android.eatery.model;
+
+import android.content.Context;
+import com.cornellappdev.android.eatery.TimeUtil;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.threeten.bp.DayOfWeek;
+import org.threeten.bp.LocalDate;
+import org.threeten.bp.LocalDateTime;
+import org.threeten.bp.LocalTime;
+import org.threeten.bp.ZoneId;
+import org.threeten.bp.ZonedDateTime;
+import org.threeten.bp.format.DateTimeFormatter;
+
+public class DiningHallModel extends EateryModel {
+
+  private Map<Integer, List<MealModel>> mWeeklyMenu;
+
+  public DiningHallModel() {
+    this.mWeeklyMenu = new HashMap<>();
+  }
+
+  public List<MealModel> getMenuForDay(DayOfWeek day) {
+    List<MealModel> val = mWeeklyMenu.get(day.getValue());
+
+    if (val != null) {
```

Comment:
inline `if` statement, this can also be 3 lines

Reply:
https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used